Makefile

```diff
@@ -161,9 +162,19 @@ docker-push:
 release:
 	@sed -i -e 's/^VERSION := .*/VERSION := ${NEWVERSION}/' ./Makefile
 
-release-manifest:
+release-manifest: update-helm-chart
 	@sed -i'' -e 's@image: $(REPOSITORY):.*@image: $(REPOSITORY):'"$(NEWVERSION)"'@' ./config/manager/manager.yaml ./deploy/gatekeeper.yaml
 
+update-helm-chart:
+	@echo "Updating chart version to: ${NEWVERSION}"
+	@sed -i "s/appVersion: .*/appVersion: ${NEWVERSION}/" ${CHART_PATH}/Chart.yaml
+	@sed -i "s/version: .*/version: ${NEWVERSION}/" ${CHART_PATH}/Chart.yaml
+
+	@echo "Updating chart images tag to: ${NEWVERSION}"
+	@sed -i -E "s/image: (.*):.*/image: \1:${NEWVERSION}/" ${CHART_PATH}/values.yaml
+	@sed -i -E "s/sidecarImage: (.*):.*/sidecarImage: \1:${NEWVERSION}/" ${CHART_PATH}/values.yaml
+	@sed -i -E "s/	image: (.*): (.*):.*/	image: \1:${NEWVERSION}/" ${CHART_PATH}/values.yaml
```

Comment:
maybe i missed this in values.yaml, but where is `sidecarImage` and why is `image` updated 2x?

Reply:
You're right -- not needed, will remove. Thanks!